src/main/java/com/amihaiemil/docker/SystemDiskSpaceInfo.java

```diff
@@ -0,0 +1,60 @@
+package com.amihaiemil.docker;
+
+import javax.json.JsonArray;
+import javax.json.JsonObject;
+
+/**
+ * Docker disk space usage information.
+ * @author Boris Kuzmic (boris.kuzmic@gmail.com)
+ * @since 0.0.6
+ */
+public final class SystemDiskSpaceInfo implements DiskSpaceInfo {
+
+    /**
+     * Response Json object from system/df call.
+     */
+    private final JsonObject json;
+
+    /**
+     * Ctor.
+     * @param jsonObject Response Json from system/df call
+     */
+    SystemDiskSpaceInfo(final JsonObject jsonObject) {
+        this.json = jsonObject;
```

Comment:
@bkuzmic After making it polymorphic, the ``jsonObject`` is passed to ``super(...)``